tools/verify_examples.sh

```diff
@@ -1,76 +1,104 @@
-#!/bin/bash
 
+#!/bin/bash
```

Comment:
No need to start the file with an empty line.

Reply:
the empty line should be before `# Copyright The OpenTelemetry Authors`